src/components/pluralize.cjsx

```diff
@@ -0,0 +1,18 @@
+React = require 'react'
+
+# TODO find a better pluralizer
```

Comment:
Sorry - I got the link wrong on that.  The one I've used is: https://github.com/jeremyruppel/underscore.inflection  (not inflections).

When I was closing the tab I noticed that the previous one hadn't been updated in two years - so don't use it.